public/static/locales/en/oauth.json

```diff
@@ -0,0 +1,15 @@
+{
+    "invalid_request": "The authorization request sent to the remote system by the DE was invalid. Please contact support.",
+    "unauthorized_client": "The DE is not authorized to request access to the remote system. Please contact support.",
+    "access_denied": "The remote system denied the authorization request. Please try again or contact support.",
+    "unsupported_response_type": "The remote system does not support the response requested by the DE. Please contact support.",
+    "invalid_scope": "The remote system does not support the scope requested by the DE. Please contact support.",
+    "server_error": "The remote system system encountered an error. Please try again or contact support.",
+    "temporarily_unavailable": "The remote system is temporarily unavailable. Please try again or contact support.",
+    "no_auth_code_provided": "Invalid authorization attempt. Please contact support.",
+    "no_state_id_provided": "Required information was missing from the authorization attempt. Please contact support.",
+    "general_service_error": "The DE service encountered an error. Please contact support.",
```

Comment:
These error codes are specific to the DE:

- `no_auth_code_provided` means that the `code` query parameter was missing from the callback URL.
- `no_state_id_provided` means that the `state` query parameter was missing from the callback URL.
- `general_service_error` means that the API call to finish the authorization process failed. In this case, we can look at the `apps` and `terrain` logs for more information.